### PR TITLE
DOC: Indicate that the normal vector points outward

### PR DESCRIPTION
**Pull request by Leengit**
This changes the text for the plane cut tool form from `Clipped output model (positive side)` to `Clipped output model (outside)` but does not change the tooltip, which is `Portion of the cut model that is on the same side of the plane as the normal.`  Similarly, `Clipped output model (negative side)` is changed to `Clipped output model (inside)`.  I am thinking that for the purposes of `Union`, `Intersection`, etc. what the user needs to know is which side is inside and which side is outside.  That these also have interpretations as negative side and positive side is less important.

If this pull request is approved, I will follow up immediately with a pull request for the documentation starting [here](https://github.com/Slicer/Slicer/blob/3802d812b1a5d855399d8a19cea0bd1705715d52/Docs/user_guide/modules/dynamicmodeler.md?plain=1#L34).

### CODE REVIEW

**Review by Sunderlandkyl — APPROVED** [review]
Seems good to me, and makes the naming consistent with vtkClipPolyData.

**Comment by lassoan** [comment]
A plane has a positive and negative side (normal vector is on the positive side). The current labels referring to "positive" and "negative" side are accurate. Why would you like to change this?

Planes have no "inside" and "outside", even when using multiple planes and various operation types. For example, how "inside" and "outside" could be interpreted in the screenshot below?

image

image

image

The only thing I don't find intuitive about this tool is the "operation type" names. I just raise this here because maybe this inconsistency leads caused the potential misunderstanding that led to this pull request.

Current behavior (Px means positive side of plane x):
- union: P1 AND P2 AND P3
- intersection: P1 OR P2 OR P3
- difference: P1 OR NOT (P2 OR P3)

I would expect this behavior:
- union: P1 OR P2 OR P3
- intersection: P1 AND P2 AND P3
- difference: P1 AND NOT (P2 OR P3)

**Comment by Leengit** [comment]
Yes, the interpretation of `union`, `intersection`, `difference` is what motivates my change.  Currently,

- Negative side of union = N1 OR N2 OR N3 (i.e., keep any point that is on the negative side of at least one plane)
- Negative side of intersection = N1 AND N2 AND N3 (i.e., keep only points that are on the negative side of all three plane)

That is, it is the negative side, both for the input (N1, N2, and N3) and for the output (union or intersection) that one should be focusing on to best understand what `union` and `intersection` mean.  To me the word `inside` says "focus here" much better than `negative side` does.  If there's a word or short phrase that says it even better than "inside" then let's go with that.

**Comment by Leengit** [comment]
@lassoan I am hoping for additional feedback from you.  Please see my comment May 17, 11:11AM EDT.

**Comment by lassoan** [comment]
Document: output is the negative side of the plane

GUI:
- Clipped output model (negative side) -> Output model (negative side)
- Clipped output model (positive side) -> Complement output model (positive side)

**Comment by jcfr** [comment]
Thanks @Leengit  and @lassoan  for working on this :pray:

**Comment by Leengit** [comment]
I just force-pushed a commit that reflects the language that we agreed upon at today's Slicer Weekly Hangout.

**Comment by Leengit** [comment]
I just added comments to the source code to reflect the changes in the user interface.

File: DynamicModeler/Logic/vtkSlicerDynamicModelerPlaneCutTool.cxx [hunk 1]
```diff
@@ -137,7 +137,7 @@ vtkSlicerDynamicModelerPlaneCutTool::vtkSlicerDynamicModelerPlaneCutTool()
 
   ParameterInfo parameterOperationType(
     "Operation type",
-    "Method used for combining the planes",
+    "Method used for combining the negative sides of the planes",
```
Comment (lassoan) [inline DynamicModeler/Logic/vtkSlicerDynamicModelerPlaneCutTool.cxx:140]:
```suggestion
    "Method used for combining the negative sides of the planes. This setting has no effect if only a single clipping plane is selected.",
```

**Review by lassoan — COMMENTED** [review]
Thank you, it all looks good to me (I only added a minor clarification to the operation type documentation).

**Comment by Leengit** [comment]
I have incorporated all the suggestions.

**Comment by Leengit** [comment]
AFAIK, this is ready to approve and merge.

**Review by lassoan — APPROVED** [review]
It all looks good. Thank you for your contribution!

**Pull request merged**